examples/test/utils.py

```diff
@@ -79,16 +79,20 @@ def update_azureml_config(olive_config):
     if workspace_name is None:
         raise Exception("Please set the environment variable WORKSPACE_NAME")
 
+    exclude_managed_identity_credential = (
+        {"exclude_managed_identity_credential": True} if "EXCLUDE_MANAGED_IDENTITY_CREDENTIAL" in os.environ else {}
```

Comment:
Is this `EXCLUDE_MANAGED_IDENTITY_CREDENTIAL` set in our pipeline variables?

Reply:
This is for local testing purposes. Azure VMs have system managed identity so it messes with the azure login. We can set this env so that it uses our azure cli login instead.

Reply:
I always add this config by myself. Probably we can add this as a note to our doc.